SWLOR.Game.Server/Perk/MartialArts/CircleKick.cs

```diff
@@ -93,10 +96,12 @@ private void ApplyFeatChanges(NWPlayer oPC, NWItem unequippingItem)
                 offHand = (new Object());
             }
 
-            if ((!mainHand.IsValid && !offHand.IsValid) ||
-                (mainType != CustomItemType.MartialArtWeapon || offType != CustomItemType.MartialArtWeapon))
+            if (mainType != CustomItemType.MartialArtWeapon)
             {
-                receivesFeat = false;
+                if (mainType != CustomItemType.None)
```

Comment:
Not every equippable "weapon" will have a custom item type. One example is a torch, should we ever add those. I think you want to stick to the original type of check where you're looking for a martial weapon and the absence of a weapon to maintain existing functionality. Otherwise items which don't fall under our custom item type checks would be treated like an unarmed weapon.